.github/actions/spelling/allow.txt

```diff
@@ -30,6 +31,7 @@ opensource
 PEET
 PWAs
 ranhpa
+ranpha
```

Comment:
Are these the same entry?

Reply:
I don't know. My addition gets [flagged as a spelling error](https://github.com/dpprdan/winget-pkgs/actions/runs/5452378176/jobs/9919806090#step:2:659).